src/main/resources/swagger/api-docs.yaml

```diff
@@ -39,155 +39,269 @@ securityDefinitions:
-#      parameters:
-#        - in: body
-#          description: create project request
-#          name: createProjectRequest
-#          required: true
-#          schema:
-#            $ref: '#/definitions/CreateRawlsBillingProjectFullRequest'
-#      tags:
-#        - Billing
-#      summary: create billing project in firecloud and google
-#      operationId: createBillingProjectFull
-#      security:
-#        - googleoauth:
-#            - openid
-#            - email
-#            - profile
-#            - https://www.googleapis.com/auth/cloud-billing
-#
-#  '/api/billing/{projectId}/members':
-#    get:
-#      responses:
-#        '200':
-#          description: Success
-#          schema:
-#            type: array
-#            items:
-#              $ref: '#/definitions/BillingProjectMember'
-#        '403':
-#          description: You must be a project owner to view the members of a project
-#        '500':
-#          description: Rawls Internal Error
-#      description: list members of billing project the caller owns
+  /api/billing:
+    post:
+      responses:
+        204:
+          description: Successfully Created Billing Project in FireCloud and Google
+        400:
+          description: FireCloud billing user must be an administrator of the billing account
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        403:
+          description: You must be an administrator of the google billing account
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        409:
+          description: project already exists in FireCloud or google
+        500:
+          description: FireCloud Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      description: create new billing project in FireCloud and google
+      parameters:
+        - in: body
+          description: create project request
+          name: createProjectRequest
+          required: true
+          schema:
+            $ref: '#/definitions/CreateRawlsBillingProjectFullRequest'
+      tags:
+        - Billing
+      summary: create billing project in FireCloud and google
+      operationId: createBillingProjectFull
+      security:
+        - googleoauth:
+            - openid
+            - email
+            - profile
+            - https://www.googleapis.com/auth/cloud-billing
+
+  /api/billing/{projectId}/members:
+    get:
+      responses:
+        200:
+          description: Success
+          schema:
+            type: array
+            items:
+              $ref: '#/definitions/BillingProjectMember'
+        403:
+          description: You must be a project owner to view the members of a project
+          schema:
+            $ref: '#/definitions/ErrorReport'
+        500:
+          description: FireCloud Internal Error
+          schema:
+            $ref: '#/definitions/ErrorReport'
+      description: list members of billing project the caller owns
```

Comment:
I don't think we need the description at this level because we have the summary below